webapp/src/components/workspaceSwitcher/workspaceSwitcher.tsx

```diff
@@ -56,6 +57,14 @@ const WorkspaceSwitcher = (props: Props): JSX.Element => {
                     }}
                 />
             }
+            {activeWorkspace &&
+                <span
+                    className='add-workspace-icon'
+                    onClick={() => history.push(`/workspace/${activeWorkspace?.id}?showEmptyCenterPanel=true`)}
```

Comment:
I don't like using a query string for this. Can we just send it to the workspace? I realize we have code in `boardPage.tsx` that loads the lastBoard and lastView if not passed. I'm in favor of either removing that code or you could set `UserSettings.lastBoardId = null` here before setting the history.

However, I don't think the code in `boardPage.tsx` is necessary any more. I believe it was there because we didn't have another way to handle it...now we do.

Reply:
Nice! Good idea. Thanks. Done